napari_animation/interpolation.py

```diff
@@ -142,10 +136,13 @@ def slerp(a, b, fraction):
         : tuple
     Interpolated Euler angles between a and b at fraction.
     """
-    key_rots = R.from_euler("ZYX", [a, b], degrees=True)
-    slerped = ExtraSlerp([0, 1], key_rots)
-    q = slerped(fraction).as_quat()
-    return quaternion2euler(q, degrees=True)
+    initial_rotation, final_rotation = [
+        R.from_euler("ZYX", eulers, degrees=True) for eulers in (a, b)
+    ]
```

Comment:
This can be written as : 
```Python 
initial_rotation, final_rotation = R.from_euler("ZYX", [a, b], degrees=True) 
```
as the `Rotation` class can behave array-like ! It's just a suggestion, if you think what you wrote helps with readability I'll merge anyways :)